packages/terra-form-radio/src/RadioField.jsx

```diff
@@ -123,7 +123,7 @@ const RadioField = (props, { intl }) => {
   );
 
   const content = React.Children.map(children, (child) => {
-    if (child.type.isRadio) {
+    if (child && child.type.isRadio) {
```

Comment:
We should update [checkboxfield](https://github.com/cerner/terra-core/blob/master/packages/terra-form-checkbox/src/CheckboxField.jsx#L126) to work the same as well. @yfaney can you update that as part of this PR as well.

Reply:
@bjankord Added: 9b8324a7a